tests/integration_tests/performance/test_snapshot_restore_performance.py

```diff
@@ -138,6 +138,7 @@ def get_snap_restore_latency(
         vcpu_count=vcpus,
         mem_size_mib=mem_size,
     )
+    basevm.memory_monitor.guest_mem_mib = mem_size
```

Comment:
Could we somehow move this to the above `basevm.machine_cfg.put` ?

Reply:
I mean, build a wrapper or something so that the burden does not fall to the test to update the size in two places. Maybe out of scope for this PR.